src/_gettsim/groupings.py

```diff
@@ -33,6 +39,61 @@ def bg_id_numpy(
     return numpy.asarray(result)
 
 
+def eg_id_numpy(
```

Comment:
Just flagging missing test coverage -- probably a fluke, but maybe this was overseen.